upload/fileinfo.go

```diff
@@ -26,6 +26,11 @@ type FileInfo struct {
 	// S3Bucket is your custom user bucket on which file are stored. Only
 	// available of you setup foreign storage bucket for your project
 	S3Bucket string `json:"s3_bucket"`
+}
+
+// GroupFileInfo holds file info in group
+type GroupFileInfo struct {
+	FileInfo
```

Comment:
Could you please add an integration test that would check that the `default_effects` field is actually returned by, for example, `upload. GroupInfo` call?